makefile

```diff
@@ -171,4 +171,19 @@ local-db-seed:
 
 local-server-tests:
 	@echo "Running tests in local app container"
-	@docker exec -it $(PROJECT_NAME)-server npm test
\ No newline at end of file
+	@docker exec -it $(PROJECT_NAME)-server npm test
+
+
+mongo-tunnel:
```

Comment:
Nit - We might not want to commit these changes

Reply:
Agreed, our bastion host was intended to be temporary anyway. We should actually coordinate with Vyas to get our readme updated with better instructions for the VPN connection, I think.

Reply:
I keep but hide all infra related info.